app/assets/stylesheets/ie8.css.scss

```diff
@@ -0,0 +1,16 @@
+/* add exemptions styling - mainly to position FRA code */
+span.exmpt {
+  font-weight: 700;
+  float:right;
+  margin-right: -80px;
+  display: block;
+  zoom: 1;
+  position:relative;
+}
+
+label.exemption {
+  background-color: #ededed;
+  border-color: #ededed;
+  padding-right: 100px;
+  width: 66.6666666667%;
+}
```

Comment:
Yeah I saw that but because this is for IE and it has been tested I was wary of changing it much. And then extracting the colours into variables would just increase the line count. I will fix the Comment and SpaceAfterPropertyColon warnings though.